lib/components/Button/index.tsx

```diff
@@ -16,7 +16,8 @@ export interface ButtonOwnProps {
   btnSize?: Size,
   btnStyle?: Style
 }
-export type ButtonProps = ButtonOwnProps & React.HTMLProps<HTMLButtonElement>;
+
+export type ButtonProps = ButtonOwnProps & any; // & React.HTMLProps<HTMLButtonElement>
```

Comment:
@magne4000 This is the typing issue I was talking about:l

```
ERROR in ./lib/components/ButtonFeedback/index.tsx
[tsl] ERROR in /Users/station-mathieu/Workspace/Station/theme/lib/components/ButtonFeedback/index.tsx(87,8)
      TS2322: Type '{ children: (ReactElement<any> | Element)[]; sheet?: any; btnSize?: Size | undefined; btnStyle?: ...' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<ButtonImpl> & Readonly<{ children?: ReactNode; }> ...'.
  Type '{ children: (ReactElement<any> | Element)[]; sheet?: any; btnSize?: Size | undefined; btnStyle?: ...' is not assignable to type 'IntrinsicClassAttributes<ButtonImpl>'.
    Types of property 'ref' are incompatible.
      Type 'string | ((instance: HTMLButtonElement | null) => any) | RefObject<HTMLButtonElement> | undefined' is not assignable to type 'string | ((instance: ButtonImpl | null) => any) | RefObject<ButtonImpl> | undefined'.
        Type '(instance: HTMLButtonElement | null) => any' is not assignable to type 'string | ((instance: ButtonImpl | null) => any) | RefObject<ButtonImpl> | undefined'.
          Type '(instance: HTMLButtonElement | null) => any' is not assignable to type 'RefObject<ButtonImpl>'.
            Property 'current' is missing in type '(instance: HTMLButtonElement | null) => any'.
```

Reply:
https://github.com/getstation/theme/pull/12/commits/ba8df7cd8b8582c08d9104de4058899feac4cc0d
for details : https://github.com/Microsoft/TypeScript/issues/5478